src/util/MongoBinaryDownload.js

```diff
@@ -91,13 +91,22 @@ export default class MongoBinaryDownload {
   }
 
   async checkMd5(mongoDBArchiveMd5: string, mongoDBArchive: string) {
+    if (
+      typeof process.env.MONGOMS_SKIP_MD5_CHECK === 'string'
+        ? ['1', 'on', 'yes', 'true'].indexOf(process.env.MONGOMS_SKIP_MD5_CHECK.toLowerCase()) !==
+          -1
+        : false
+    ) {
+      return undefined;
+    }
     const signatureContent = fs.readFileSync(mongoDBArchiveMd5).toString('UTF-8');
     const m = signatureContent.match(/(.*?)\s/);
     const md5Remote = m ? m[1] : null;
     const md5Local = md5File.sync(mongoDBArchive);
     if (md5Remote !== md5Local) {
       throw new Error('MongoBinaryDownload: md5 check is failed');
     }
+    return undefined;
```

Comment:
Why do we need to return an undefined here ?

Reply:
@AJRdev if you add somewhere in the function body `return ` statement, then eslint doesn't tolerate implicit `return undefined`

Reply:
We can make improve signature `async checkMd5(mongoDBArchiveMd5: string, mongoDBArchive: string): Promise<?boolean>` and return at the end a `true` value. 

At line 100 keep `undefined`, cause we do not check md5 and not sure "true or false" is it.

Reply:
@nodkz sure, we'll do it